src/v3/src/components/Form/Layout.tsx

```diff
@@ -26,17 +35,67 @@ import renderers from './renderers';
 // eslint-disable-next-line import/no-cycle
 import Stepper from './Stepper';
 
-type LayoutProps = {
-  uischema: UISchemaLayout;
-};
-
 const getElementKey = (element: UISchemaElement, index: number): string => {
   const defaultKey = [element.type, element.key, index].join('_');
   return element.type === 'Field' && (element as FieldElement).key
     ? [(element as FieldElement).key].join('_')
     : defaultKey;
 };
 
+type LayoutProps = {
+  uischema: UISchemaLayout;
+};
+
+type StyledAccordionProps = {
+  uischema: AccordionLayout;
+};
+
+const StyledAccordionSummary = styled((props: AccordionSummaryProps) => (
+  // eslint-disable-next-line react/jsx-props-no-spreading
+  <AccordionSummary {...props} />
+))(({ theme }) => ({
+  padding: 0,
+  width: 'fit-content',
+  '& .MuiAccordionSummary-content': {
+    // marginLeft: theme.spacing(1),
+    margin: 0,
+    color: theme.palette.primary.main,
+    '&:hover': {
+      textDecoration: 'underline',
+      textDecorationColor: theme.palette.primary.main,
+    },
+  },
+}));
+
+const Accordion: FunctionComponent<StyledAccordionProps> = ({ uischema }) => {
```

Comment:
Can these Accordion elements be moved into their own component file?

Reply:
I initially started down that road, the issue I ran into was that I needed a way to re-use existing elements (Heading, Paragraph, List, etc). Without creating custom rendering logic (similar to what exists in renderers.tsx) inside the accordion component, there was not a good way (that I could think of) to implement it. 

Did you have any other ideas?

The biggest need is the ability to compose other elements inside the accordion body without duplicating logic or creating a bunch of custom logic in this new component.

Reply:
Oh, wait, I think I misunderstood 😅 

You mean just moved to a separate file similar to Stepper? If so, is disabling the cyclic dependency lint rule ok?

Reply:
Yeah, I was just thinking a separate file, since it's mostly a standalone component. I'm a little bit worried about non-type related cyclic dependencies, though. What is the component in the cycle?

Reply:
Its the `Layout` Component, the Accordion uses `Layout` and `Layout` uses `Accordion` :(